### PR TITLE
UX: Hide full screen after user input

### LINKED ISSUE #7643: Cards are hidden behind the toolbar and become hard to read (fail to adjust the card location when users toggle the full screen)
###### Reproduction Steps

1. fresh install and create some cards in some deck
2. enable "Preferences" -> "Gestures" -> "Enable gestures"
3. chose "Preferences" -> "Reviewing" -> "Fullscreen mode" -> "Hide the system bars and answer buttons"
4. open the deck and review the cards 
5. scroll down to show the toolbar and answer buttons (now the cards are hidden behind the toolbar)
6. click the answer buttons (do not return back to the full screen), the cards are still hidden behind the toolbar (difficult for users
to read the cards)

###### Expected Result

Under the above full screen mode, (1) if the user scrolls down to show the toolbar and answer buttons when reviewing cards,
anki should *automatically* adjust the location of cards (e.g., showing the cards under the toolbar), and keep this location until the user returns back to the full screen mode; (2) when the user explicitly returns back to the full screen mode, anki should adjust the location of cards back again (e.g., showing the cards at the location of the original toolbar).

###### Actual Result

Under the above full screen mode, anki fails to adjust the location of cards. As a result, the user cannot easily review cards.

###### Debug info

This issue is reproducible on the latest release version (2.13.5) and the latest develop version (2.14alpha29), and is reproducible on Android emulator (Android 7.1) and Huawei nova 5 (Android 9.0).

The reproducing video:

found_issue_in_v2 14alpha14_v2 14alpha16

###### Research
*Enter an [x] character to confirm the points below:*

- [X] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] I have searched for similar existing issues here and on the user forum
- [X] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by david-allison-1**
## Purpose / Description
It was frustrating to undo a card while in full screen, and the menu bars did not auto-hide.

This was the same for answering or flipping cards

## Fixes
Fixes #7643 

## Approach
Add a delayed hide call

## How Has This Been Tested?

My device & unit tests

## Learning (optional, can help others)
DelayedHide did not work with a 100ms delay when an options item was selected

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java [hunk 1]
```diff
@@ -974,9 +976,16 @@ public void fillFlashcard() {
     public void displayCardQuestion() {
         // show timer, if activated in the deck's preferences
         initTimer();
+        delayedHide(100);
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java:979]: is 100 sufficient here while 1000 was needed elsewhere? same question below
If so, then LGTM

**Review by mikehardy — APPROVED** [review]
LGTM pending question on 100 vs 1000 in all methods vs just 1

**Comment by david-allison-1** [comment]
> LGTM pending question on 100 vs 1000 in all methods vs just 1

Yep - Tapping a button on the app bar was different than tapping a button in the content area. 

Presumably the 100ms fires before the "tap undo" click event fires.

File: AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java (hunk 1 above)
Comment (david-allison-1) [reply AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java:979]: Yep - Tapping a button on the app bar was different than tapping a button in the content area. 

Presumably the 100ms fires before the "tap undo" click event fires.

Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java:979]: excellent - shipping time then :-), merge at will

Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java:979]: actually CI had finished, I just merged it :)

**Pull request merged**